proto/api/v1/compliance_service.proto

```diff
@@ -109,4 +117,12 @@ service ComplianceService {
             get: "/v1/compliance/aggregatedresults"
         };
     }
+
+    rpc UpdateCompliance(UpdateComplianceRequest) returns (Empty) {
```

Comment:
The function name is misleading. I recommend changing the name to `UpdateComplianceStandardConfig`